docs/sphinx/examples/python/dynamics/squeezing.py

```diff
@@ -0,0 +1,97 @@
+import cudaq
+from cudaq import spin, Schedule, ScipyZvodeIntegrator
+
+import numpy as np
+import cupy as cp
+import matplotlib.pyplot as plt
+import os
+
+# Set the target to our dynamics simulator
+cudaq.set_target("dynamics")
+
+# In this example, we simulate a two-level system that decays into a squeezed vacuum state.
+# The master equation is given in "The theory of open quantum systems", by `Francesco Petruccione and Heinz-Peter Breuer`, section 3.4.3 - 3.4.4.
+
+
+def n_thermal(w: float, w_th: float):
+    """
+    Return the number of average photons in thermal equilibrium for a
+        an oscillator with the given frequency and temperature.
+    """
+    if (w_th > 0) and np.exp(w / w_th) != 1.0:
+        return 1.0 / (np.exp(w / w_th) - 1.0)
+    else:
+        return 0.0
+
+
+# Problem parameters
+w0 = 1.0 * 2 * np.pi
+gamma0 = 0.05
```

Comment:
This again comes from QuTiP; even the comments are the same https://nbviewer.org/urls/qutip.org/qutip-tutorials/tutorials-v5/lectures/Lecture-12-Decay-into-a-squeezed-vacuum-field.ipynb

Reply:
removed